core/lib/ReactCore_Internal.re

```diff
@@ -1538,16 +1198,31 @@ module Make = (OutputTree: OutputTree) => {
       key != component.key ? {...component, key} : component;
     Flat(Element(componentWithKey));
   };
+
   let listToElement = l => Nested(l);
 
-  module RemoteAction = {
-    type t('action) = {mutable act: 'action => unit};
-    let actDefault = _action => ();
-    let create = () => {act: actDefault};
-    let subscribe = (~act, x) =>
-      if (x.act === actDefault) {
-        x.act = act;
-      };
-    let act = (x, ~action) => x.act(action);
+  /* Temporary mechanism for keeping identity without the first class module */
```

Comment:
What ideas do you have in mind to replace this mechanism? Would it make sense to open an issue to explain or discuss about them?